src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.html

```diff
@@ -57,7 +57,7 @@
                       formControlName="fs_name"
                       name="fs_name"
                       id="fs_name"
-                      (change)="pathChangeHandler()">
+                      (change)="callBoth()">
```

Comment:
nit: lets rename it to understand what its doing ? like what its calling ?
e.g `handleVolume()`, `volumeChangeHandler()`

Reply:
Done, thanks!